Automattic/jetpack
# Slideshow block: Remove duplicate class names to allow proper block selection

Fixes https://github.com/Automattic/jetpack/issues/37675

## Proposed changes:

* This PR removes the `className` from the returned Slideshow component. This prevents the duplication of the following classnames, which resulted in the block only being selectable if clicking the pause/play icon or the dot-pager section: `block-editor-block-list__block wp-block is-selected aligncenter wp-block-jetpack-slideshow`
* The PR also includes a small CSS tweak to make sure the pause/play button remains visible with the class name tweak.

### Other information:


## Jetpack product discussion

https://github.com/Automattic/jetpack/issues/37675

## Does this pull request change what data or activity we track or use?

No.

## Testing instructions:

* To replicate, on trunk (or using the Beta tester plugin and making sure to use the Release Candidate) create a post and add a Slideshow block.
* Regardless of whether auto-play is enabled, you won't be able to select the block by clicking on any of the images within the block. Only clicking the pause/play icon or the dot-pager section will select the block.
* To test the fix, apply this PR and refresh the post. You should now be able to click anywhere within the block in the editor and the block should be selected.
* This should also be the same for Atomic and simple sites.
